modules/primer-navigation/lib/underline-nav.scss

```diff
@@ -0,0 +1,79 @@
+.UnderlineNav {
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid $gray-200;
+}
+
+.UnderlineNav-items {
+  display: flex;
+  margin-bottom: -1px;
+}
+
+.UnderlineNav-item {
+  padding: $spacer-3 $spacer-2;
+  margin-right: $spacer-3;
+  font-size: $body-font-size;
+  line-height: $lh-default;
+  color: $text-gray;
+  text-align: center;
+  border-bottom: 2px solid transparent;
+
+  .Counter {
+    margin-left: $spacer-1;
+  }
+
+  .octicon {
+    margin-right: $spacer-1;
+    color: $gray-400;
+  }
+
+  &:hover,
+  &:focus {
+    color: $text-gray-dark;
+    text-decoration: none;
+    border-bottom-color: $gray-300;
+    transition: 0.2s ease;
+
+    .octicon {
+      color: $gray-500;
+    }
+  }
+
+  &.selected {
+    font-weight: $font-weight-bold;
+    color: $text-gray-dark;
+    border-bottom-color: $orange-600;
+
+    .octicon {
+      color: $gray-500;
+    }
+  }
+}
+
+.UnderlineNav--right {
+  justify-content: flex-end;
+
+  .UnderlineNav-item {
+    margin-right: 0;
+    margin-left: $spacer-3;
+  }
+
+  .UnderlineNav-actions {
+    flex: 1 1 auto;
+  }
+}
+
+.UnderlineNav-actions {
+  align-self: center;
+}
+
+.UnderlineNav--full {
+  display: block;
+
+  .container-lg,
+  .container-md,
+  .container-xl {
```

Comment:
Same here re nesting other Primer styles. I`UnderlineNav-container` may work? It's a little bit of a quirky one and potentially could be done with utilities but I can image this tripping people up so probably good to offer it as part of the component styles.

Reply:
Changed this to `.UnderlineNav-container` 👍